package.json

```diff
@@ -40,7 +42,7 @@
     "redux-form": "^7.0.3"
   },
   "peerDependencies": {
-    "@folio/stripes": "^2.0.0",
+    "@folio/stripes": "^2.3.0",
```

Comment:
that looks good, but maybe there is a possibility to use a newer version, https://github.com/folio-org/stripes/blob/master/doc/stripes-framework.md#upgrading-to-v26